_tutorials/new-component/create-a-new-component-part-8.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Create A New Origami Component - Part 8 Component Lifecycle
+description: A step-by-step tutorial which teaches you how to build and deploy a new Origami component.
+cta: Learn how to create an Origami component
+collection_listing_display: false
+
+---
+
+# {{page.title}}
+
+The "Create A New Origami Component" tutorial is split into eight parts and is intended to be followed sequentially from start to finish:
+- [part one](/docs/tutorials/create-a-new-component-part-1/): Intro & Boilerplate
+- [part two](/docs/tutorials/create-a-new-component-part-2/): Base Styles
+- [part three](/docs/tutorials/create-a-new-component-part-3/): Themes & Brands
+- [part four](/docs/tutorials/create-a-new-component-part-4/): Demos
+- [part five](/docs/tutorials/create-a-new-component-part-5/): JavaScript
+- [part six](/docs/tutorials/create-a-new-component-part-6/): Testing
+- [part seven](/docs/tutorials/create-a-new-component-part-7/): Documentation
+- part eight: Component Lifecycle
+
+In part eight we will learn how to publish our component to the Origami registry 🎉, and discuss the lifecycle of a published component.
+
+_We don't actually want to publish an example component `o-example`. If you have been following along so far using `o-example`, rather than working on your own component that should actually be published, read this part of the tutorial as a reference only until you're ready to publish a new component for real._
+
+## Push To Github
+
+The first step of publishing our component to the [Origami Registry](https://registry.origami.ft.com/components/) is to commit our work and push to a new Github repository under the [Financial-Times](https://github.com/Financial-Times/) organisation (or other [supported Github organisation](/spec/v1/components/#source-control)). There is a [new Github repository tutorial](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-new-repository) which might be helpful if you have never created a remote Github repository before. Note Origami repositories are usually public and open-source unless there is a reason not to be (for example [o-fonts-assets](https://github.com/Financial-Times/o-fonts-assets/) is private due to our font license).
+
+When your new component is pushed to Github [update access in the Github settings of your repository](https://docs.github.com/en/github/getting-started-with-github/access-permissions-on-github). Grant these teams the following permissions:
```

Comment:
What do we think about using a `.github/settings.yml` file which controls the repo settings? If we have that, then this section of the tutorial doesn't _have_ to exist.

The file does not work on it's own, we would need either a GitHub app or a GitHub Action -- https://github.com/probot/settings

Reply:
Scratch that. As discussed in Slack: Looks like developers don't have the right Github access to add repos to the probot app. Customer Products get around that with [Financial-Times/manage-github-apps](https://github.com/Financial-Times/manage-github-apps) but then that's maybe getting more complicated to explain that what we have 😅